src/Normalizer/ContentEntityNormalizer.php

```diff
@@ -249,9 +251,45 @@ public function denormalize($data, $class, $format = NULL, array $context = arra
       }
     }
 
+    // Remove changed info, otherwise we can get validation errors.
+    if (isset($data['changed'])) {
+      unset($data['changed']);
+    }
+
     // @todo Move the below update logic to the resource plugin instead.
     $storage = $this->entityManager->getStorage($entity_type_id);
 
+    // Denormalize entity reference fields.
+    foreach ($data as $field_name => $field_info) {
+      if (!is_array($field_info)) {
+        continue;
+      }
+      foreach ($field_info as $delta => $item) {
+        if (isset($item['entity_type_id']) && isset($item['target_uuid'])) {
+          $target_storage = $this->entityManager->getStorage($item['entity_type_id']);
+          $target_entity = $target_storage->loadByProperties(array('uuid' => $item['target_uuid']));
+          $target_entity = !empty($target_entity) ? reset($target_entity) : NULL;
+          if ($target_entity) {
+            $data[$field_name][$delta] = array(
+              'target_id' => $target_entity->id(),
+            );
+            continue;
+          }
+          $target_entity_values = array('uuid' => $item['target_uuid']);
+
+          if ($target_storage instanceof TermStorageInterface) {
+            $target_entity_values['vid'] = 'tags';
+            $target_entity_values['name'] = 'Stub name for taxonomy term';
+          }
```

Comment:
I think we should execute a hook here, something like `hook_entity_create_stub()`. Then we should move the logic that is specific to terms into a hook implementation in `relaxed.module`.